src/Filter/ScssFilter.php

```diff
@@ -51,4 +54,130 @@ public function input($filename, $input)
         $return = $this->_runCmd($bin, '', array('PATH' => $this->_settings['path']));
         return $return;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getDependencies(AssetTarget $target)
+    {
+        $children = [];
+        foreach ($target->files() as $file) {
+            $imports = CssUtils::extractImports($file->contents());
+            if (empty($imports)) {
+                continue;
+            }
+
+            $ext = $this->_settings['ext'];
+            $extLength = strlen($ext);
+
+            $deps = [];
+            foreach ($imports as $name) {
+                if (preg_match('/(.*)\/\*\*\/\*$/', $name, $matches)) {
+                    $path = $this->_findFile($matches[1]);
+                    $relPathBegin = mb_strlen($path) - mb_strlen($matches[1]);
+                    $result = [];
+                    $this->_dirToArray($path, $relPathBegin, $result);
+                    $deps = array_merge($deps, $result);
+                    continue;
+                }
+                $pathinfo = pathinfo($name);
+                if (substr($pathinfo['basename'], 0, 1) !== '_') {
+                    $name = $pathinfo['dirname'] . '/_' . $pathinfo['basename'];
+                }
```

Comment:
This looks like a change that will break existing applications that do not have `_` prefixes.